packages/mjml-carousel/README.md

```diff
@@ -34,7 +34,6 @@ This component enables you to set the styles of the carousel elements.
 attribute | unit | description | default value
 ----------|------|-------------|---------------
 align | string | horizontal alignment | center
-background-color | string | column background color | none
```

Comment:
This should be rename as `container-background-color` and added into allowedAttributes instead !